### ISSUE TITLE
Always use alt attributes on images

### ISSUE THREAD

**Issue by alexander-dubinski** [Milligan123/TeaCozy#3]
https://github.com/Milligan123/TeaCozy/blob/735f7c054ac359e8605b3aa5f3cc2a3d1740878d/index.html#L13

Best practices and even laws related to webpage accessibility in the U.S. dictate that informative alt tags should be used with images
